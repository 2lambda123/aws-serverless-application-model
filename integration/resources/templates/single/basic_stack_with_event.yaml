Resources: 
  NestedStack: 
    Type: AWS::CloudFormation::Stack
    Properties: 
      TemplateURL: ${stackurl}

  ThumbnailFunction:
    Type: AWS::Serverless::FunctionReference
    Properties:
      FunctionName:
        Fn::GetAtt:
          - NestedStack
          - Outputs.FunctionName
      Events:
        ImageBucket:
          Type: S3
          Properties:
            Bucket:
              Ref: Images
            Events: s3:ObjectCreated:*    
                  
  Images:
    Type: AWS::S3::Bucket       