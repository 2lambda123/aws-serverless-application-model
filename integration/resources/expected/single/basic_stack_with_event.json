[
    { "LogicalResourceId":"Images", "ResourceType":"AWS::S3::Bucket" },
    { "LogicalResourceId":"NestedStack", "ResourceType":"AWS::CloudFormation::Stack" },
    { "LogicalResourceId":"ThumbnailFunctionImageBucketPermission", "ResourceType":"AWS::Lambda::Permission" }
]